rocketmq-knative/source/README.md

```diff
@@ -0,0 +1,75 @@
+## RocketMQ Source
+[![License](https://img.shields.io/badge/license-Apache%202-4EB1BA.svg)](https://www.apache.org/licenses/LICENSE-2.0.html)
+[![Language](https://img.shields.io/badge/Language-Go-blue.svg)](https://golang.org/)
+
+
+## Overview
+
+RocketMQSource which is a separate Kubernetes custom resource fires a new event each time an event is published on an Apache RocketMQ Cluster.
+
+## Quick Start
+
+### build  RocketMQ Source
+
+1 Clone the project on your Kubernetes cluster master node:
+
+```
+$ git clone https://github.com/apache/rocketmq-externals
+
+$ cd rocketmq-externals/rocketmq-knative/source
+```
+
+2 Download dependent files:
+
+```
+$ go get github.com/wlliqipeng/vendor
+$ cp -r $GOPATH/src/github.com/wlliqipeng/vendor  rocketmq-externals/rocketmq-knative/source
```

Comment:
Do not leave your directory here.

Reply:
Go get command will automatically create the github.com/wlliqipeng directory under $GOPATH/src

Reply:
Ok，I will delete my directory

Reply:
Done